runtime/build.rs

```diff
@@ -52,7 +52,16 @@ fn main() {
         "cargo:rerun-if-changed={}",
         pre_commit_hook_path.to_string_lossy()
     );
-    let enabled_hooks_dir = root_path.join(".git/hooks");
+    // Flexible path to support linked worktrees
```

Comment:
Please, add more detailed description, it is hard to catch an issue. I would add that associated worktrees share git hooks with the main worktree and thus hooks should be installed into the main one. Probably it worth to add a link to git docs.